addon/components/uni-modal.js

```diff
@@ -19,11 +21,14 @@ export default Component.extend({
 
   // This observer is used to bypass the scroll on mobile when a modal is open
   onOpenChangeObserver: observer('isOpen', function() {
-    let overflowClass = 'overflow-hidden';
-
-    this.get('isOpen') ? $('body').addClass(overflowClass) : $('body').removeClass(overflowClass);
+    this.get('isOpen') ? $('body').addClass(OVERFLOW_HIDDEN_CLASS) : $('body').removeClass(OVERFLOW_HIDDEN_CLASS);
   }),
 
+  didDestroyElement() {
+    this._super(...arguments);
+    $('body').removeClass(OVERFLOW_HIDDEN_CLASS);
```

Comment:
New line before, and I think it would be cool to make the `overflowClass` customizable :)